pages/art.md

```diff
@@ -3,4 +3,11 @@
 ## [How to Draw a Rat](https://www.deviantart.com/nezupanda/art/Rat-Tutorial-290900060)
 [Archived Link](https://web.archive.org/web/20240627000346/https://www.deviantart.com/nezupanda/art/Rat-Tutorial-290900060)
 
-A small visual tutorial on how to draw a rat.
+A small  tutorial on how to draw a rose.
```

Comment:
It looks like you accidentally deleted the description for the old link. Please fix this. Also please pull the latest updates to resolve any conflicts